packages/core/src/components/TextField/TextField.types.ts

```diff
@@ -0,0 +1,14 @@
+export type TextFieldTextType =
```

Comment:
If it's not too much trouble I think it would be better to be called TextFieldType, but not critical

Reply:
Done